roles/jd-server/src/main.rs

```diff
@@ -102,10 +105,13 @@ async fn main() {
     let url = config.core_rpc_url.clone() + ":" + &config.core_rpc_port.clone().to_string();
     let username = config.core_rpc_user.clone();
     let password = config.core_rpc_pass.clone();
+    let (submit_solution_sender, submit_solution_receiver): (Sender<String>, Receiver<String>) =
+        unbounded();
```

Comment:
Is there a specific reason to not use a bounded channel?

Reply:
Ok.
Can you confirm that if create the channel like this
`let (submit_solution_sender, submit_solution_receiver): (Sender<String>, Receiver<String>) = bounded(3);`
then at most three `SubmitSolution` are sent? Sorry for the trivial question but better to ask before that getting in trouble after ^^

Reply:
no is the max number of message that can be "inside"  the channel check the documentation if is not clear

Reply:
added `boubnded(10)` as it is done similarly to other parts of the codebase